src/components/fields/password-field/password-field.js

```diff
@@ -72,16 +73,30 @@ class PasswordField extends React.Component {
     return (
       <Constraints.Horizontal constraint={this.props.horizontalConstraint}>
         <Spacings.Stack scale="xs">
-          <FieldLabel
-            title={this.props.title}
-            hint={this.props.hint}
-            description={this.props.description}
-            onInfoButtonClick={this.props.onInfoButtonClick}
-            hintIcon={this.props.hintIcon}
-            badge={this.props.badge}
-            hasRequiredIndicator={this.props.isRequired}
-            htmlFor={this.state.id}
-          />
+          <Spacings.Inline>
```

Comment:
```suggestion
          <Spacings.Inline alignItems="center">
```

Reply:
We should probably use it with `justify-content: space-between` in this case (see #552)